### DISCUSSION TITLE
How to fix it?

### DISCUSSION THREAD

**Discussion by Sensei-Developer** [ROM-builders/temporary#14734] [Q&A]
Why the last two committees of your clang, they broke everything for me and others, how to fix it?

https://cirrus-ci.com/build/4703944952774656

@Apon77

**Comment by RahulPalXDA** [comment]
> Why the last two committees of your clang, they broke everything for me and others, how to fix it?
> 
> https://cirrus-ci.com/build/4703944952774656
> 
> @Apon77

You can try changing clang. Like move to proton clang etc. Not sure it'll fix that but you can try.

**Reply by Alone0316** [reply]
He didn't break it, someone is cloning that clang while building Los which leaded to the problem
